src/tests/GfxReplayTest.cpp

```diff
@@ -108,11 +125,23 @@ TEST(GfxReplayTest, recorder) {
   ASSERT(keyframes[0].stateUpdates[0].first == instanceKey);
 
   // verify frame #1 has our translation and semantic Id
-  ASSERT(keyframes[1].stateUpdates.size() == 1);
+  ASSERT(keyframes[1].stateUpdates.size() == 2);
```

Comment:
Can you add a comment explaining why we expect 2 state updates on this keyframe? The reason is because (1) we moved `node`, and (2) we created `node2`, which gets a stateUpdate to capture its starting transform after creation.